runtime/common/src/weights/currencies.rs

```diff
@@ -44,7 +44,7 @@ use frame_support::{
 };
```

Comment:
line 34, `pallet_currencies` instead of `orml_currencies`

(It is frustrating that in 2022, in the most widely used collaboration tool we are not able to comment on collapsed lines. Or am I missing something? :D)

Reply:
Yes, I know, but I intentionally didn't rename it because it is auto-generated output. But we can rename it, no problem. Re; commenting on collapsed lines, I think it makes sense, because it's not clear what are you intentions. You might want to add the same comment to each line, or just at the begging/end.